app/views/share/emails/_summary_row.slim

```diff
@@ -5,4 +5,4 @@ tr.shares-editor__summary-row id="share_summary_#{share.name}_#{share.id}"
   td colspan=3 = simple_format share.body
   td.shares-editor__button-cell = render 'share/btn_edit_share'
   td.shares-editor__button-cell
-    = button_to 'Delete', page_share_email_path(@page.id, share.id), {form_class: 'shares-editor__delete-variant', class: 'btn btn-danger', method: :delete, remote: true}
```

Comment:
Since the only difference between these is `page_share_#{share_type}_path`, is there a way we could extract these out into a partial?